src/anomalib/data/base/depth.py

```diff
@@ -3,7 +3,7 @@
 
 from abc import ABC
 
-import albumentations as A
+import albumentations as A  # noqa: N812
```

Comment:
Maybe we can keep it as `import albumentations`. And same for `torch.functional`

Reply:
I would also prefer that to be honest. Wanted to get your opinion

Reply:
settled then? :)